src/components/RegistrationProblem/styles.scss

```diff
@@ -0,0 +1,100 @@
+@import '../../styles/colors.scss';
+
+@font-face {
```

Comment:
It was decided in the last meeting to use Fjalla One font, so this can be deleted. I merged changes from development where it is now the automatic style in defaults.scss for H1, so you don't have to specify it.

Reply:
Thx, wasn't quite sure if this one was fjalla one too. Fixed it